src/sources/dnstap/mod.rs

```diff
@@ -67,6 +68,9 @@ pub struct DnstapConfig {
     /// Maximum number of frames that can be processed concurrently.
     pub max_frame_handling_tasks: Option<u32>,
 
+    /// Whether to downcase all DNSTAP hostnames received for consistency
+    pub lowercase_hostnames: Option<bool>,
```

Comment:
I think having this be an `Option` makes the default unclear in the docs. I realize that there is already some precedent here for optional boolean configuration values like `multithreaded`, but I think it might be clearer to just have this be a `bool` with a default value so that the docs render it (I'd likely also suggest that we migrate `multithreaded` and the other `Options` here too non-`Option` types with a default in the future).

Reply:
Maybe these can be updated in a separate PR, to keep this one focused on this option. I will update this to default to `false`.